src/sentry/snuba/referrer.py

```diff
@@ -456,6 +456,7 @@ class Referrer(Enum):
     API_STARFISH_MOBILE_STARTUP_TOTALS = "api.starfish.mobile-startup-totals"
     API_STARFISH_MOBILE_SCREENS_METRICS = "api.starfish.mobile-screens-metrics"
     API_STARFISH_MOBILE_SCREENS_SCREEN_TABLE = "api.starfish.mobile-screens-screen-table"
+    API_EXPLORE_SPANS_SAMPLES_TABLE = "api.explore.spans-samples-table"
```

Comment:
this is in the "mobile starfish" section?

Reply:
sorted it alphabetically